packages/design-system/tslint.json

```diff
@@ -0,0 +1,80 @@
+{
```

Comment:
I guess it's just copied, however setting up `tslint` for this package is not worth it and we should directly setup the `eslint` linter 👍🏼 

accordingly wouldn't add it to the github action